engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -551,8 +551,8 @@ public void dispose() {
         renderableWorld.dispose();
         worldProvider.dispose();
         // TODO: Shift this to a better place, after a RenderGraph class has been implemented.
-        // TODO: Call ResetDefaultInstance() for every StateChange after it is a method of StateChange.
-        SetViewportToSizeOf.ResetDefaultInstance();
+        // TODO: Call disposeDefaultInstance() for every StateChange after it is a method of StateChange.
```

Comment:
By this, I mean that we ideally should have a disposeDefaultInstance() for every single StateChange, rather than just for SVTSO (most of them will probably be empty for now). Will make it possible to call dispose() for every single StateChange/Node.
Does that sound good?